<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="ver_seguinte">
       <property name="text">
        <string>ver seguinte</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cascata">
       <property name="text">
        <string>cascata</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>514</x>
      <y>330</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="fechar_tudo">
       <property name="text">
        <string>fechar tudo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ver_janelas">
       <property name="text">
        <string>Ver janelas</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="ver_anterior">
       <property name="text">
        <string>ver anterior</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mosaico">
       <property name="text">
        <string>mosaico</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QMdiArea" name="mdiArea">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>321</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ssssssssss&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="sub">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="windowTitle">
      <string>Subwindow</string>
     </property>
    </widget>
    <widget class="QWidget" name="sub1">
     <property name="windowTitle">
      <string>Subwindow</string>
     </property>
    </widget>
    <widget class="QWidget" name="sub_2">
     <property name="windowTitle">
      <string>Subwindow</string>
     </property>
    </widget>
   </widget>
   <zorder>mdiArea</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>verticalSpacer</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
